client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -471,12 +471,11 @@ export const SeoForm = React.createClass( {
 						status="is-warning"
 						showDismiss={ false }
 						text={ translate(
-							'SEO settings are disabled because the ' +
-							'site visibility is not set to Public.'
+							"SEO settings aren't recognized by search engines while your site is Private."
 						) }
 					>
 						<NoticeAction href={ generalTabUrl }>
-							{ translate( 'View Settings' ) }
+							{ translate( 'View privacy settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **67** times:
`translate( 'Privacy Settings' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).